src/inject/core/pauseAfter.js

```diff
@@ -0,0 +1,21 @@
+import { remote } from 'electron';
+import _ from 'lodash';
+
+// we throttle this function because of a bug in gmusic.js
+// ratings are received multiple times here in a couple of ms
+// to avoid play pause to be called 5+ times we throttle it to 500ms max
+window.addEventListener('load', _.throttle(() => {
+  remote.getGlobal('PlaybackAPI').on('change:song', () => {
+    if (remote.getGlobal('PlaybackAPI').currentPauseAfter()) {
+      try {
+        GPM.playback.playPause();
+        remote.getGlobal('PlaybackAPI')._setPauseAfter(false);
+      } catch (err) {
+// calling GPM throws a ReferenceError for whatever reason, but it works, so...
+        if (err.name !== 'ReferenceError') {
```

Comment:
GPM throws a reference error because you put this in the wrong folder.  If you put it in the `GPMInject` folder it will only be injected into the Google Play Music webview not every single rendering process